packages/integration-functions/common/src/azure-service-bus/service-bus.js

```diff
@@ -0,0 +1,45 @@
+const { ServiceBusClient } = require('@azure/service-bus');
+const { DefaultAzureCredential } = require('@azure/identity');
+
+/**
+ * todo: potentially extend/reuse class from back-office/aapd 313
+ * however will listed building messages need the message transformations used there?
+ * @param {String} serviceBusHostname
+ * @param {String} topicName
+ * @param {Array.<Object>} messages
+ */
+async function sendMessageBatch(serviceBusHostname, topicName, messages) {
```

Comment:
I know the documentation is awful, but possibly this might work across different types of queue output function context.extraOutputs.set(queueOutput, ['message 1', 'message 2']); If not this code looks fine to me.

Reply:
if I get some time I'll look into how/if this works